etna/docker-compose.yml

```diff
@@ -20,6 +21,8 @@ services:
       - .:/app:cached
       - etna-app-gems:/app/vendor/bundle:rw
       - ../docker/etna-base/entrypoints:/entrypoints:ro
+      - /etc/passwd:/etc/passwd:ro
```

Comment:
Allows the etna polyphemus command to run as the current user and keep user info in that shell environment.